go/keymanager/api/grpc.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/oasislabs/oasis-core/go/common/cbor"
+	enclaverpc "github.com/oasislabs/oasis-core/go/runtime/enclaverpc/api"
+)
+
+var (
+	// Make sure this always matches the appropriate method in
+	// `keymanager-runtime/src/methods.rs`.
+	getPublicKeyRequestMethod = "get_public_key"
+
+	// requestSkipPolicyCheck defines if policy check is needed for the request.
+	requestSkipPolicyCheck = func(req interface{}) bool {
+		r, ok := req.(*enclaverpc.CallEnclaveRequest)
+		if !ok {
+			return false
+		}
+
+		// Check if policy access is needed.
+		skipPolicyCheck, err := payloadSkipPolicyCheck(r.Payload)
+		if err != nil {
+			return false
+		}
+
+		return skipPolicyCheck
+	}
+
+	// Service is the Keymanager enclave gRPC service.
+	Service = enclaverpc.New(ModuleName, requestSkipPolicyCheck)
+)
+
+// This is the Go analog of the Rust RPC Frame from runtime/src/rpc/types.go.
+type frame struct {
```

Comment:
It may be useful to expose this publicly in `api.go` (e.g., for other consumers that may want to peek into frames).